api/contract/cli.yml

```diff
@@ -81,6 +81,10 @@ paths:
     $ref: "./openapi/src/common/paths/dbrps.yml"
   "/dbrps/{dbrpID}":
     $ref: "./openapi/src/common/paths/dbrps_dbrpID.yml"
+  /authorizations:
```

Comment:
This will combine with the default server URL to become `/api/v2/authorizations` (and similar for below). The V1 auth system uses `/private/legacy/authorizations` as its endpoint prefix.

We can fix it by declaring custom `servers` for these 2 routes (see `/health` at the top for an example). I'd suggest doing it like this:
```
/legacy/authorizations:
  servers:
    - /private
```
We could also try having `/private/legacy` be the server and `/authorizations` be the operation, but I think that'll collide with V2 auth when we add it.